app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ContactsController < ApplicationController
+
+  def create
+    @contact = Contact.new(contact_params)
+    @company = Company.find(params[:company_id])
+    @contact.company_id = @company.id
```

Comment:
This is not the ideal way to create objects that are associated to each other. Because a company has many contacts, we can do...

```ruby
@contact = @company.contacts.new(contact_params)
```

we want to leverage our associations to help us :thumbsup: